src/java.desktop/share/classes/sun/awt/FontConfiguration.java

```diff
@@ -1249,10 +1251,16 @@ public String getFileNameFromPlatformName(String platformName) {
      * search path.
      */
     public String getExtraFontPath() {
+        if (head == null) {
+            throw new RuntimeException("Fontconfig head is null, check your fonts or fonts configuration");
```

Comment:
Does it make sense to create a String constant for the error message?

I understand that there's only one instance of the message in the class file, it's more to avoid duplication and to avoid having different error messages in case someone updates only one of them.